scripts/bnsd/genesis_app_state.json

```diff
@@ -31,14 +31,85 @@
       "name": "Let the Phoenix arise"
     }
   ],
-  "gconf": {
-    "cash:minimal_fee": "0.01 CASH",
-    "cash:collector_address": "0000000000000000000000000000000000000000"
+  "conf": {
+    "cash": {
+      "minimal_fee": "0.01 CASH",
+      "collector_address": "0000000000000000000000000000000000000000"
+    },
+    "migration": {
+      "admin": "cond:multisig/usage/0000000000000001"
```

Comment:
Future note, in 0.16 you can simplify this with
`seq:multisig/usage/1`
where `seq:` is like `cond:`, except it takes a normal int and encodes it as 8-byte bigendian, rather than you manually writing out the hex definition... which gets uglier when there are many.